src/components/Groups/GroupPage/PostExportAllModal.tsx

```diff
@@ -0,0 +1,225 @@
+          <Transition.Child
+            as={Fragment}
+            enter="ease-out duration-300"
+            enterFrom="opacity-0"
+            enterTo="opacity-100"
+            leave="ease-in duration-200"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+          >
+            <Dialog.Overlay className="fixed inset-0 bg-gray-500 bg-opacity-75 dark:bg-gray-900 dark:bg-opacity-75 transition-opacity" />
+          </Transition.Child>
+
+          <span
+            className="hidden sm:inline-block sm:align-middle sm:h-screen"
+            aria-hidden="true"
+          >
+            &#8203;
+          </span>
+          <Transition.Child
+            as={Fragment}
+            enter="ease-out duration-300"
+            enterFrom="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+            enterTo="opacity-100 translate-y-0 sm:scale-100"
+            leave="ease-in duration-200"
+            leaveFrom="opacity-100 translate-y-0 sm:scale-100"
+            leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+          >
+            <div className="inline-block align-bottom bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:align-middle sm:max-w-lg sm:w-full">
+              <div className="bg-white px-4 pt-5 pb-4 sm:p-6 sm:pb-4">
+                <div className="sm:flex sm:items-start">
+                  <div className="mt-3 text-center sm:mt-0 sm:ml-4 sm:text-left">
+                    <Dialog.Title
+                      as="h3"
+                      className="text-lg leading-6 font-medium text-gray-900"
+                    >
+                      Export All Posts
+                    </Dialog.Title>
+                    <div className="mt-2 text-gray-500">
+                      Please select the groups you would like to export all
+                      posts to.
+                      <div className="block">
+                        {groups.isSuccess &&
+                          (groups.data && groups.data.length > 0 ? (
+                            groups.data.map(group =>
+                              group &&
+                              group.ownerIds.includes(firebaseUser!.uid) ? (
+                                <div key={group.id}>
+                                  <label className="inline-flex items-center">
+                                    <input
+                                      type="checkbox"
+                                      className="form-checkbox"
+                                      onChange={() =>
+                                        handleGroupExportChange(group)
+                                      }
+                                    />
+                                    <span className="ml-2 text-gray-500">
+                                      {group.name}
+                                    </span>
+                                  </label>
+                                </div>
```

Comment:
minor, but I think right now it will allow you to export a group to itself, which probably isn't desireable.